MapboxNavigation/PollyVoiceController.swift

```diff
@@ -97,6 +97,8 @@ public class PollyVoiceController: RouteVoiceController {
         }
         
         switch (langCode, countryCode) {
+        case ("da", _):
+            input.voiceId = .naja
```

Comment:
Project-OSRM/osrm-text-instructions#208 adds Danish support for voice instruction generation on the server. Unfortunately, we have our own little version of [dependency hell](https://en.wikipedia.org/wiki/Dependency_hell): if this PR goes in before that the OSRM Text Instructions change makes it into the Directions API, then English sentences will be read in a Danish voice. On the other hand, if the Directions API is updated before this PR lands, then Danish instructions will be read in an English voice. We‘ll need to coordinate both the client-side and server-side changes to avoid suboptimal behavior.

Reply:
@bsudekum, what do you think about splitting out this PollyVoiceController change into a separate PR that we can merge when Project-OSRM/osrm-text-instructions#208 is deployed in the Directions API? (If #617 lands first, then the PR would become moot.)